pkg/lib/oauth/handler/handler_authz.go

```diff
@@ -381,6 +382,11 @@ func (h *AuthorizationHandler) doHandle(
 		return nil, err
 	}
 
+	err = h.validateUIImplementation(client, r)
```

Comment:
This validation should not be performed here. Instead, it should be performed in pkg/auth/handler/api/authenticationflow_v1_create.go and pkg/auth/handler/webapp/authflow_controller.go

In the two places I mention, OAuthSession is read. OAuthSession may now contain x_authentication_flow_group which be validated.

Reply:
Actually authflow controller already has the checking as well. I added this extra logic so that sdk will receive this error code before webapp is opened. 

Maybe I remove this extra checking instead then.